### PR TITLE
Add communes names to zipcode indices details

### LINKED ISSUE #193: Add gemeinde name as searchable detail to zipcodes
See discussion https://github.com/geoadmin/mf-geoadmin3/issues/2160

Add langtext (commune name) to detail https://github.com/geoadmin/service-sphinxsearch/blob/master/conf/search.conf.part#L151

@ponceta Could you do it?

### PR DESCRIPTION
**Pull request by ponceta**
Fix #193

### CODE REVIEW

File: conf/search.conf.part [hunk 1]
```diff
@@ -148,7 +148,7 @@ source src_zipcode : src_swisssearch
         SELECT \
         gid as id \
         , os_uuid::text as feature_id \
-        , plz::text as detail \
+        , coalesce(langtext,'')||' '||plz::text as detail \
```
Comment (ponceta) [inline conf/search.conf.part]: I do not really need coalesce() right?

**Comment by gjn** [comment]
also use remove accents, as above in addresses or gg25.

**Comment by ponceta** [comment]
BEFORE
image

AFTER
image

**Comment by ponceta** [comment]
This will break browserstack search test and is a bit confusing if you search bern

image

<3 Switzerland <3

**Comment by ponceta** [comment]
With @loicgasser we do not think that's a good idea. On the principle yes but this will break too much things (tests / rankings / ... )

To be thought again. @gjn what do you think?

Test on [dev](http://mf-geoadmin3.dev.bgdi.ch/)

**Comment by loicgasser** [comment]
Yes, this is more complicated than that. We'll have to alter the rank if we want to change that. People are gonna freak out for sure. Can you revert the indices on test please, so that tests don't fail tonight...

**Comment by gjn** [comment]
We don't do it. It's a hassle that's not worth it.

Users should be able to use `find` service on the npa layer, no?

**Comment by gjn** [comment]
@ponceta Can you please revert it on sphinx test instance?
(Tests fail because of it)

**Comment by ponceta** [comment]
Reverted on test

**Comment by oterral** [comment]
Query tool forever !!!!!

**Pull request closed without merging**